cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -91,4 +91,19 @@ package object cwl extends TypeAliases {
     import wdl4s.cwl.Implicits.enumerationEncoder
     commandLineTool.asJson
   }
+
+  def encodeCwlWorkflow: Workflow => Json = { workflow =>
```

Comment:
I wrote it to look like the decodeCwl method. But actually, a simple one-argument method may be nicer.